example/src/repository/medicine/MedicineRepository.ts

```diff
@@ -1,12 +1,12 @@
-import { BaseRepositoryMysql, IRepositoryGeneric, Metadata } from 'attiv';
```

Comment:
Metadata wasn't referenced?

Reply:
No, Metadata is used to send messages to queues. In this case wasn't being used.